.github/dependabot.yml

```diff
@@ -5,6 +5,9 @@ updates:
     schedule:
       interval: monthly
     open-pull-requests-limit: 10
+    ignore:
```

Comment:
Could you add a comment here about why we need to ignore these?

Reply:
Added, and moved over multiaddr too.